subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/transport/StubTransport.kt

```diff
@@ -0,0 +1,27 @@
+package com.avito.android.test.report.transport
+
+import com.avito.android.test.report.ReportState
+import com.avito.android.test.report.model.TestMetadata
+import com.avito.filestorage.FutureValue
+import com.avito.filestorage.RemoteStorage
+
+internal object StubTransport : Transport {
+
+    override fun sendReport(state: ReportState.Initialized.Started) {
+    }
+
+    override fun sendContent(
+        test: TestMetadata,
+        request: RemoteStorage.Request,
+        comment: String
+    ): FutureValue<RemoteStorage.Result> {
+        return FutureValue.create(stubError(request))
```

Comment:
this signals that API has problems
We should do not create stub errors

Reply:
yes, i have plans to remove all async work in reportsr ; file operations will be blocking

see MBS-11110